airflow/dags/create_external_tables/gtfs_rt_v2/rt_v2_service_alerts.yml

```diff
@@ -1,5 +1,10 @@
 operator: operators.ExternalTable
 bucket: "{{ env_var('CALITP_BUCKET__GTFS_RT_PARSED') }}"
+post_hook: |
+  SELECT *
+  FROM `{{ get_project_id() }}`.external_gtfs_rt_v2.service_alerts
+  WHERE dt = '{{ ds }}'
```

Comment:
Will these error/fail if no data is returned...? I can query dates with no data, it just says there's no data. 

![image](https://user-images.githubusercontent.com/55149902/191618580-1cb6fdeb-09e8-45eb-8505-f046b4dc32f0.png)

Reply:
I guess if you're saying it won't be an effective check, the other option would be have it select the prior day's data. But I actually think because of the execution time shenanigans we were talking about yesterday the data will definitely be there, because execution date is "yesterday" relative to when this runs....?

Reply:
yeah I was thinking about whether the hook would actually be exercising the schema at all; we can discuss more tomorrow but maybe we just need to fetch the latest `dt` in the operator and use it as the query parameter

FWIW templating SQL in this way could technically lead to injection though `{{ ds }}` is trusted so it's fine

Reply:
We discussed offline & agreed it's better to just kill the post hooks entirely, because of the potentially fragile date logic and it's not convenient to just get one row (i.e., performance may be bad/wasteful)